ush/run_fv3.ksh

```diff
@@ -0,0 +1,39 @@
+#!/bin/ksh --login
+ 
+# Set the queueing options 
+#SBATCH --nodes=60 --ntasks-per-node=4   ## refer to the run_fcst task section in the workflow xml
+#SBATCH -t 6:40:00
+#SBATCH -A nrtrr
+# #SBATCH -qos debug
+#SBATCH -J fv3_fcst
+#SBATCH --partition=sjet  ## check which jet partition was used from the run_fcst log file in the cycle that needs to be repeated
+ 
+set -x
+#-------------------------------------------
+# point to the source code for executable and
+# for loading the same modules used in the run that needs to be repeated
+#-------------------------------------------
+SRCDIR=/WHERE-YOUR-ufs-srweather-app-CODE-IS/dev4-ufs-srweather-app
+module purge
+module use $SRCDIR/regional_workflow/modulefiles/tasks/jet
+module load run_fcst
+ 
+FV3_EXE=$SRCDIR/exec/fv3_gfs.x
+
+#-------------------------------------------
+# the workdir is a hard copy of the cycle that needs to be repeated, for example, /lfs4/BMC/nrtrr/NCO_dirs/stmp/tmpnwprd/RRFS_dev4/2021010612
+# remove the IC/LBC source (HRRRX/, RAPX/), forecast files (dynf*.nc, phyf*.nc), log files (logf*), post and restart files (if applicable) 
+# must keep all the fix files and INPUT/ 
+#-------------------------------------------
```

Comment:
Should the script do this cleanup, or no?

Reply:
The script can do the part for removing  the forecast files etc. But for IC/LBC source, it could be RAPX/HRRRX or FV3GFS.

Reply:
Now I think it again, for RRFS_devs, it is RAP/HRRRX or RAP/HRRR since we don't use FV3GFS for the input.
Do you prefer I include the clean-up in the script? I can add that part, if it is prefered.

Reply:
Would it hurt to remove all of them with a `-f` so it doesnt't complain? ;)

Reply:
It doesn't hurt to do so but it depends on the target cycle how many / which files/directories exist and therefore what files/directories to remove. So the key point here is to have all the fix files (preferred to be hard copied with "cp -L" ) and INPUT/

Reply:
I added one line to remove the existing forecast files and log files. Users can decide whether to remove extra stuff to free some space, if applicable.